### PR TITLE
Fix issue #63 add alertDialog before signout

### PR DESCRIPTION
**Pull request by snufflesrea**
#63 
screenshot from 2018-06-15 13-13-50

### CODE REVIEW

File: Android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -149,4 +149,9 @@
     Also known as Bahai Temple, Lotus Temple is one of the finest crafted structures is in Delhi</string>
     <string name="empty_description">No description available.</string>
 
+    <!--signout AlertDialog-->
+    <string name="signout_alert_messsage">Are you sure want to Signout?</string>
```
Comment (Swati4star) [inline Android/app/src/main/res/values/strings.xml]: change 'Signout' to 'sign out'

**Comment by Swati4star** [comment]
Just incorporate one comment I added, and squash the commits into a single one.
Rest looks good 👍

**Comment by Swati4star** [comment]
You might need to create a new PR. The changed files count is 65.

**Comment by snufflesrea** [comment]
Yeah. Later I will look at it. It seems I messed up something

**Pull request closed without merging**